src/showcase/stories/angular/chip/chip-group/chip-group.stories.ts

```diff
@@ -0,0 +1,92 @@
+import { SbbChip } from '@sbb-esta/lyne-angular/chip/chip';
+import { SbbChipGroup } from '@sbb-esta/lyne-angular/chip/chip-group';
+import { SbbFormField } from '@sbb-esta/lyne-angular/form-field/form-field';
+import { Args, Meta, moduleMetadata } from '@storybook/angular';
+import { ArgTypes, InputType } from '@storybook/types';
+
+const readonly: InputType = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Input field',
+  },
+};
+
+const size: InputType = {
+  control: {
+    type: 'inline-radio',
+  },
+  options: ['s', 'm', 'l'],
+  table: {
+    category: 'Form field',
+  },
+};
+
+const hiddenLabel: InputType = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Form field',
+  },
+};
+
+const floatingLabel: InputType = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Form field',
+  },
+};
+
+const argTypes: ArgTypes = {
+  readonly,
+  size,
+  hiddenLabel,
+  floatingLabel,
+};
+
+const args: Args = {
+  disabled: false,
+  readonly: false,
+  negative: false,
+  size: 'm',
+  hiddenLabel: false,
+  floatingLabel: false,
+};
+
+const meta: Meta = {
+  decorators: [
+    moduleMetadata({
+      imports: [SbbFormField, SbbChip],
+    }),
+  ],
+  title: 'elements/sbb-chip/sbb-chip-group',
+  component: SbbChipGroup,
+  argTypes,
+  args,
+  render: ({ negative, size, hiddenLabel, floatingLabel, disabled, readonly }: Args) => ({
+    props: { negative, size, hiddenLabel, floatingLabel, disabled, readonly },
+    template: `
+      <sbb-form-field
+        [negative]=${negative}
+        size=${size}
+        [hidden-label]=${hiddenLabel}
+        [floating-label]=${floatingLabel}
```

Comment:
EG. [src/showcase/stories/angular/autocomplete/autocomplete.stories.ts](https://github.com/sbb-design-systems/lyne-angular/blob/d71bff98962a99251d2a631dfbf201ac25d7f6ed/src/showcase/stories/angular/autocomplete/autocomplete.stories.ts#L169)
```suggestion
        [negative]="negative"
        [size]="size"
        [hidden-label]="hiddenLabel"
        [floating-label]="floatingLabel"
```